src/Network/AWS/Flow.hs

```diff
@@ -36,6 +37,7 @@ import           Safe
 
 register :: MonadFlow m => Plan -> m ()
 register Plan{..} = do
+  logInfoN "event=register\n"
```

Comment:
Hmmm, so I see this here, but not for other flow events (decide, execute, etc.)?